pkg/resourcemanager/mysql/mysqlhelper.go

```diff
@@ -9,9 +9,9 @@ import (
 	"context"
 	"database/sql"
 	"fmt"
+	"github.com/hashicorp/go-multierror"
```

Comment:
This seems like a weird file to have included here? And I think the change you have here is actually not the recommended ordering anyway (it seemed correct before?)